src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js

```diff
@@ -10,24 +10,41 @@ import createGuid from "../../../__internal__/utils/helpers/guid";
  *
  */
 
-export const tokensClassName = `sageDesignTokens-${createGuid()}`;
+const activeThemes = {};
+
+const carbonInstanceId = createGuid();
+
+const snakeCase = (input) =>
```

Comment:
I think this const might be confusing as this is snake_case and this is kebab-case. Calling this const `kebabCase` maybe better suited to the format that the function returns.

Reply:
Of course. That was just a brain fart ;)